CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+# 1.2.7
+
+## Enhancements
+* Duplicated `externalId` are now allowed for upserts. If duplicated
+`externalId` fields are used for upserts
```

Comment:
> Duplicated `externalId` are now allowed for upserts

one `externalId` __is__, several `externalId`s __are__

Although the formatting when pluralizing `code snippet`s looks really weird :/

> If duplicated `externalId` fields are used for upserts

I don't understand this sentence

Reply:
Unfinished sentence :D Good catch, thanks.